code/jobs/CloneGitRepo.php

```diff
@@ -52,7 +52,6 @@ public function perform() {
 		}
 
 		fwrite($fh, sprintf('[%s] Cloning repository %s to %s', date('Y-m-d H:i:s'), $repo, $path) . PHP_EOL);
-		echo "[-] CloneGitRepo starting" . PHP_EOL;
```

Comment:
It echo's out to the page which is an issue because I'm trying to return json data.